tests/functional/t/too/too_many_arguments.py

```diff
@@ -36,3 +36,7 @@ def func_call():
 # +1: [too-many-arguments]
 def name1(param1, param2, param3, /, param4, param5, *args, param6="apple", **kwargs):
     return param1, param2, param3, param4, param5, param6, args, kwargs
+
+
+# Negative case, see `_check_isinstance_args` in `./pylint/checkers/typecheck.py`
+isinstance(5, int, int) # [too-many-arguments]
```

Comment:
Can we add a case checking that both `too-many-arguments` and `isinstance-second-argument-not-valid-type` can be emitted on the same line?

Reply:
Sure! Is it fine to have such a case live in `too_many_arguments.py`, considering it crosses multiple warnings?

Reply:
Yeah, that strikes me as fine since it's still unit-testing the too_many_arguments check, to ensure it doesn't early return.